driverconfig/config.yaml

```diff
@@ -196,75 +196,75 @@ csiSideCars:
   - name: attacher
     images:
       - version: v121
-        tag: k8s.gcr.io/sig-storage/csi-attacher:v3.5.0
+        tag: k8s.gcr.io/sig-storage/csi-attacher:v4.0.0
       - version: v122
-        tag: k8s.gcr.io/sig-storage/csi-attacher:v3.5.0
+        tag: k8s.gcr.io/sig-storage/csi-attacher:v4.0.0
```

Comment:
Please revert this change for v122 as we are not supporting K8s 1.22
Please do the same for all sidecars below

Reply:
Taken care